systems-management/nisysmgmt.yml

```diff
@@ -0,0 +1,2030 @@
+        description: Required if hostname is not provided. A string representing the IP address of the system to register.
+        example: 1.1.1.1
+      password:
+        type: string
+        description: A string representing the password to use access the system.
+        example: test
+      username:
+        type: string
+        description: A string representing the username to use access the system.
+        example: root
+
+  RegisterSystemsResponse:
+    title: Register Systems Response
+    description: Response of a request to register systems.
+    type: object
+    properties:
+      request_id:
+        description: The ID of the registration request.
+        type: string
+        example: 068530b9-b059-4c0e-9820-9e9cc4e0832c
+    required: [request_id]
+
+  UnregisterSystemsRequest:
+    title: Unregister Systems Request
+    description: Request to unregister systems.
+    type: object
+    properties:
+      tgt:
+        description: Array of string representing the IDs of systems to be removed.
+        type: array
+        items:
+          type: string
+          example: VMware_Virtual_Platform--SN-VMware-56_4d_30_26_4f_c9_5c_05-b7_68_d1_cd_99_6e_22_16--MAC-02-01-02-03-04-08
+      force:
+        description: Specify as true to remove systems from the database immediately, or false to wait until the unregister job returns from systems. If true, unregister job failures will not be cached.
+        type: boolean
+        default: true
+        example: false
+    required: [tgt]
+
+  UnregisterSystemsResponse:
+    title: Unregister Systems Response
+    description: Response of the request to unregister systems.
+    type: object
+    properties:
+      jid:
+        description: The job ID of the unregistration job sent to connected systems.
+        type: string
+        example: 20190211135234448026_28320
+      removed_ids:
+        description: An array of system IDs removed from the database immediately.
+        type: array
+        items:
+          type: string
+          example: VMware_Virtual_Platform--SN-VMware-56_4d_30_26_4f_c9_5c_05-b7_68_d1_cd_99_6e_22_16--MAC-02-01-02-03-04-08
+      failed_ids:
+        description: An array of objects defining failed unregister operations.
+        type: array
+        items:
+          type: object
+          title: Failed Unregister Request
+          properties:
+            id:
+              description: The ID of the system on which unregister operation failed.
+              type: string
+              example: VMware_Virtual_Platform--SN-VMware-56_4d_30_26_4f_c9_5c_05-b7_68_d1_cd_99_6e_22_16--MAC-02-01-02-03-04-08
+            error:
+              $ref: '#/definitions/Error'
+
+  CancelJobRequest:
+    description: Object containing the job ID and a system ID.
+    title: Cancel Job Request
+    type: object
+    properties:
+      jid:
+        description: The job ID of the job to cancel.
+        type: string
+        example: 20190211135234448026_28320
```

Comment:
20190211135234448026_28320 gets rendered in scientific notation as "jid": 2.019021113523445e+24 which is probably not what was intended.